source/.gitignore

```diff
@@ -14,3 +14,7 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+
+# vim's stuff
```

Comment:
I'm actually kind against putting 'my work environment garbage files' lines in the .gitignore for a project. I think a better solution is to keep extraneous files related to your development tools in a global gitignore file
```
git config --global core.excludesfile ~/.gitignore_global
```